packages/react-components/react-tabs/src/components/Tab/Tab.types.ts

```diff
@@ -23,6 +23,10 @@ export type TabSlots = {
   content: NonNullable<Slot<'span'>>;
 };
 
+export type TabInternalSlots = TabSlots & {
+  contentReservedSpace?: Slot<'span'>;
```

Comment:
I don't think this should be moved to be a slot.  We are using the existing content but bolded to get an exact pixel match for taking up space between selected and non-selected tabs. It only happens when the tab content is a string and we probably shouldn't inject an additional span element.

Reply:
According to the snapshots it seems you're already injecting an additional span element. There's no modification in the DOM from the before and after this implementation